nise/generators/aws/s3_generator.py

```diff
@@ -42,10 +41,7 @@ def __init__(self, start_date, end_date, payer_account, usage_accounts, attribut
 
     def _get_arn(self, avail_zone):
         """Create an amazon resource name."""
-        arn = "arn:aws:ec2:{}:{}:snapshot/snap-{}".format(
-            avail_zone, self.payer_account, self.fake.ean8()
-        )  # pylint: disable=no-member
-        return arn
+        return "arn:aws:ec2:{}:{}:snapshot/snap-{}".format(avail_zone, self.payer_account, self.fake.ean8())
```

Comment:
I think this line is wrong.

Should it be:

```suggestion
        return "arn:aws:s3:{}:{}:snapshot/snap-{}".format(avail_zone, self.payer_account, self.fake.ean8())
```